pyknos/mdn/mdn.py

```diff
@@ -160,7 +160,35 @@ def log_prob(self, inputs: Tensor, context=Optional[Tensor]) -> Tensor:
         """
 
         logits, means, precisions, sumlogdiag, _ = self.get_mixture_components(context)
+        return self.log_prob_mog(inputs, logits, means, precisions, sumlogdiag)
+
+    @staticmethod
+    def log_prob_mog(
+        inputs: Tensor,
+        logits: Tensor,
+        means: Tensor,
+        precisions: Tensor,
+        sumlogdiag: Tensor,
+    ) -> Tensor:
+        """
+        Return the log-probability of `inputs` under a MoG with specified parameters.
+
+        Unlike the `log_prob()` method, this method is fully detached from the neural
+        network and is a `staticmethod`.
```

Comment:
would remove the "and is a static method", this is clear from the code, no? 
Instead I would add why this is useful, e.g., add sth like "can be used independent of the neural net in case the MoG parameters are already known"?